scripts/version/create-snippet.js

```diff
@@ -26,8 +27,11 @@ const encoding = 'base64';
 const inputText = fs.readFileSync(inputPath, 'utf-8');
 const integrity = algorithm + '-' + crypto.createHash(algorithm).update(inputText).digest(encoding);
 const outputText = header + snippet(getName(), integrity, getVersion());
+const { code: transpiledOutputText } = babel.transformSync(outputText, {
+  presets: ['env'],
+});
```

Comment:
added babel transpilation step here. guarantees that output snippet file uses es5 syntax.